Integrations/src/main/java/io/deephaven/integrations/learn/Input.java

```diff
@@ -67,9 +68,9 @@ private Input(String[] colNames, Function<Object[], Object> gatherFunc) {
     }
 
     /**
-     * Gets the caller for the given gather function.
+     * Gets the gather function passed to Input.
```

Comment:
```suggestion
     * Gets the gather function.
```
In many cases, it is not the function which is passed in, because you are doing a PyFunctionCaller.